src/import_edf.jl

```diff
@@ -204,22 +208,49 @@ can be either a `String` giving the generated channel name, or a `Pair` mapping
 a canonical name to a list of alternatives that it should be substituted for
 (e.g., `"canonical_name" => ["alt1", "alt2", ...]`).
 
+`unit_alternatives` lists standardized unit names and alternatives that map to them.
+See `OndaEDF.STANDAR_UNITS` for defaults.
+
+`preprocess_labels(label::String)` is applied to raw edf signal header labels
+beforehand; defaults to `identity`.
+
 See `OndaEDF.STANDARD_LABELS` for the labels (`signal_names => channel_names`
 `Pair`s) that are used to extract EDF signals by default.
 
 """
-function extract_channels_by_label(edf::EDF.File, signal_names, channel_names)
+function extract_channels_by_label(edf::EDF.File, signal_names, channel_names; unit_alternatives=STANDARD_UNITS, preprocess_labels=identity)
     matcher = x -> begin
         # yo I heard you like closures
-        return s -> match_edf_label(s.header.label, signal_names,
-                                    x isa Pair ? first(x) : x,
-                                    channel_names)
+        return s -> begin
+            m = match_edf_label(preprocess_labels(s.header.label),
+                                signal_names,
+                                x isa Pair ? first(x) : x,
+                                channel_names)
+            !isnothing(m) && return m
+            # channel info is sometimes misplaced in transducer_type field; only accept these if corresponding signal_name occurs in channel label
+            m = match_edf_label(preprocess_labels(s.header.transducer_type),
+                                signal_names,
+                                x isa Pair ? first(x) : x,
+                                channel_names)
+            isnothing(m) && return m
+            lowercase_label = _safe_lowercase(s.header.label)
+            any(occursin(name, lowercase_label) for name in signal_names) && return m
+            return nothing
+        end
```

Comment:
fix isnothing check and lift out/comment on the x isa Pair thing:

```suggestion
        # x is either a channel name (string), or a channel_name => alternatives Pair.
        this_channel_name = x isa Pair ? first(x) : x
        return s -> begin
            m = match_edf_label(preprocess_labels(s.header.label),
                                signal_names,
                                this_channel_name,
                                channel_names)
            !isnothing(m) && return m
            # channel info is sometimes misplaced in transducer_type field; only accept these if corresponding signal_name occurs in channel label
            m = match_edf_label(preprocess_labels(s.header.transducer_type),
                                signal_names,
                                this_channel_name,
                                channel_names)
            !isnothing(m) && return m
            lowercase_label = _safe_lowercase(s.header.label)
            any(occursin(name, lowercase_label) for name in signal_names) && return m
            return nothing
        end
```